tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/ballerinahelper/Message.java

```diff
@@ -81,26 +81,43 @@ public static void getPayload(BallerinaASTModelBuilder ballerinaASTModelBuilder,
     /**
      * Set the payload of a message
      */
-    public static void setPayload(BallerinaASTModelBuilder ballerinaASTModelBuilder, Map<String, Object> parameters) {
+    public static void setPayload(BallerinaASTModelBuilder ballerinaASTModelBuilder, Map<String, Object> parameters,
```

Comment:
Can you please check how feasible to use Map<String, String> in here? I couldn't find any places where we used any object references other than String. We might able to avoid unnecessary object casting to String.